recipes/dectate/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
```

Comment:
Please take a look at the example recipe:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml

and attempt to conform this recipe as closely as possible to that guide.